### PR TITLE
Add verify banner

### PR DESCRIPTION
**Pull request by marukohao**
- [x] Describe the purpose of the change
add verify banner component in dashboard alert history view

- [x] Create test cases for your changes if needed
Screenshot 2024-02-21 at 9 35 59 AM

- [ ] Include the ticket id if possible (Collaborator only)

### CODE REVIEW

File: packages/notifi-dapp-example/public/logos/discord-icon.svg
Comment (eric-notifi) [inline packages/notifi-dapp-example/public/logos/discord-icon.svg:1]: We might want to remove this if it is already in sprite.

File: packages/notifi-dapp-example/.gitignore
Comment (eric-notifi) [inline packages/notifi-dapp-example/.gitignore:1]: I think we need a rebase so that it wont become entire files changes against main

**Review by eric-notifi — CHANGES_REQUESTED** [review]

File: packages/notifi-dapp-example/src/components/VerifyBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+export type VerifyBannerProps = Readonly<{
+  unVerifiedDestinations: ReadonlyArray<FormField>;
+  setCardView: (cardView: CardView) => void;
+}>;
+
+export const VerifyBanner: React.FC<VerifyBannerProps> = ({
+  unVerifiedDestinations,
+  setCardView,
+}: VerifyBannerProps) => {
+  const unVerifiedDestinationsString = useMemo(() => {
```
Comment (eric-notifi) [inline packages/notifi-dapp-example/src/components/VerifyBanner.tsx:15]: If we are going to have `useTarget` hook, we might want to also move this one to `useTarget` hook and rename it accordingly (maybe `unVerifiedTargets`?)

File: packages/notifi-dapp-example/src/assets/Icon.tsx [hunk 2]
```diff
@@ -16,7 +16,8 @@ export type SpriteIconId =
   | 'email-icon'
   | 'slack-icon'
   | 'telegram-icon'
-  | 'discord-icon';
+  | 'discord-icon'
```
Comment (eric-notifi) [inline packages/notifi-dapp-example/src/assets/Icon.tsx:19]: I still can see the following files in `public/logo/`, shall we remove those?
- discord-icon.svg
- email-icon.svg
- telegram-icon.svg
- slack-icon.svg

**Review by eric-notifi — CHANGES_REQUESTED** [review]
Seems like the github action fail to run, might want to check it out (package-lock.json issue?)

File: packages/notifi-dapp-example/src/components/VerifyBanner.tsx (hunk 1 above)
Comment (marukohao) [reply packages/notifi-dapp-example/src/components/VerifyBanner.tsx:15]: will do in next pr with dashboard destination section

**Pull request merged**